README.md

```diff
@@ -82,6 +82,15 @@ Below is a table showing ``relative_start_pos`` (`start`) and ``relative_end_pos
 | ``<i> ... </i><b></b>`` | 0 | 0 | 1 | 2 |
 | ``<i> ... <b></b></i>`` | 0 | 2 | 0 | 1 |
 
+## Pre-processing Metadata:
+
+### Entity Tags.
+
+**Pre-requisite steps for preprocessing Entity Tags**
+
+* Run `pip install -r bsmetadata/vendor/REL/requirements.txt`
+* Run `preprocessing_scripts/download_entity_processing_files.sh` to download all the required files.
+
```

Comment:
It would also be perfect if we could indicate how much space it would take to make this download (this can be added in a future PR)